content/en/api/logs/list_logs.md

```diff
@@ -0,0 +1,37 @@
+---
+title: List Logs
+type: apicontent
+order: 16.2
+external_redirect: /api/#list-logs
+---
+
+## List Logs
+
+List endpoint returns logs that match a log search query. Results are paginated.
```

Comment:
👋 don't we want to document the response payload as well?

Reply:
I mean on top of the example provided. Maybe this is how we document all REST APIs, sorry if it's a dummy question :)

Reply:
plop, result payload are provided: https://github.com/DataDog/documentation/pull/4026/files#diff-0d2646143c4ec52a12bff5d13d59a500 but maybe you had something different in mind?

Reply:
Yes I would have thought we could specifically say what each field is and what value it can contain when that makes sense (for instance the different status values, how the "nextLogId" can be used...). And this on top of one or several example(s).

Reply:
That makes sense, but it would mean doing it for all API endpoints :/ I think we are going to wait for API v2 before documenting all returned fields. 
Regarding `nextLogid`, there is now a small doc in `how to use pagination` that explains how to do it. For the status, idk what it refers to... is it the request status? ping @pcariou-dd